state_machine.py

```diff
@@ -488,6 +492,11 @@ def _enter(self, machine):
         else:
             machine.intervalForInactivityEventS = MAX_INACTIVITY_TIME_S
 
+    def _exit(self, machine):
+        pass
+
+    def _update(self, machine):
+
         machine.system.sensor.poll_sensors()
```

Comment:
A function in the system class would be a cleaner way of doing this

Reply:
see above comment